lib/katello/permissions/host_permissions.rb

```diff
@@ -26,7 +26,8 @@
   'katello/api/v2/hosts_bulk_actions/content_overrides',
   'katello/api/v2/hosts_bulk_actions/environment_content_view',
   'katello/api/v2/hosts_bulk_actions/release_version',
-  'katello/api/v2/hosts_bulk_actions/traces',
+  'katello/api/v2/hosts_bulk_actions/fetch_traces',
```

Comment:
This controller method and route should be renamed back to 'traces' from 'fetch_traces'

Reply:
Updated